src/Cryptol/Prelude.hs

```diff
@@ -32,3 +32,4 @@ arrayContents = B.pack [there|lib/Array.cry|]
 
 cryptolTcContents :: String
 cryptolTcContents = [there|lib/CryptolTC.z3|]
+
```

Comment:
I'm guessing this is one of those make-a-trivial-change-to-get-cabal-to-recompile situations?

Reply:
yup